.gitlab-ci.yml

```
@@ -178,3 +184,18 @@ deploy_to_debugger_backend:
     UPSTREAM_BRANCH: $CI_COMMIT_BRANCH
     UPSTREAM_TAG: $DEPLOY_TAG
 
+
+package:
+  extends: .package
+  rules:
+    - if: $DOTNET_PACKAGE_VERSION
+      when: on_success
+    - if: '$CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+\.[0-9]+(-prerelease)?$/'
+      when: on_success
+  script:
+    - ../.gitlab/build-deb-rpm.sh
+
+.release-package:
+  stage: deploy
+  variables:
+    PRODUCT_NAME: auto_inject-dotnet
```

```suggestion
    PRODUCT_NAME: dotnet
```
If I'm correct, `PRODUCT_NAME` is used to retrieve the version from a file named ${PRODUCT_NAME}.version (see [here](https://github.com/DataDog/auto_inject/blob/29699c938f49c9bd184ba99767419d298b85797a/gitlab/template/packaging.yml#L69))

On the release-pipeline triggered we have a [failure](https://gitlab.ddbuild.io/DataDog/dd-trace-dotnet/-/jobs/286509157) and a missing version when trying to promote the package 
```
/usr/bin/bash: line 158: auto_inject-dotnet.version: No such file or directory
```

---

Thanks for raising the issue. Fixed in 55f3beebf2159c36e751cd513be065740f592613
The [new run](https://gitlab.ddbuild.io/DataDog/dd-trace-dotnet/-/jobs/288460516) LGTM